pkgs/tools/typesetting/sile/default.nix

```diff
@@ -109,6 +109,7 @@ stdenv.mkDerivation rec {
       such as InDesign.
     '';
     homepage = "https://sile-typesetter.org";
+    changelog = "https://github.com/sile-typesetter/sile/raw/v${version}/CHANGELOG.md";
```

Comment:
The Changelog commit should be part of the sile update commit. You can still keep the credit to @marsam with:

https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors

Reply:
Sure, done.